cypress/e2e/calendar/createCalendarEvent.feature

```diff
@@ -0,0 +1,35 @@
+@stable_test
+@regression_test
+Feature: Calendar - create a simple event in the calendar
+
+  As a user, I want to be able to create a simple event in the calendar without course/team and check if it's displayed properly.
+
+
+  Scenario Outline: create an event in the calendar as an admin, teacher and student
+    Given I am logged in as a '<user>' at '<namespace>'
+    When I go to calendar overview
+    Then I see calendar page
+    When I click on icon Next
```

Comment:
do we need to click on the icon Next? is this step required or can directly click on day 18 on the first page?

Reply:
Yes, it is required, because I want to select ay defined day (the 18th) and if the 18th of the current month is already in the past, this is not possible. So I go to the next month.